### PR TITLE
Fix #19: Support replicates

### LINKED ISSUE #19: Adjust interface to allow specification of W and H
If I run the nnmf function and it fails to converge, I should be able to rerun it with the current state of W and H rather than having to start from scratch. The current function interface does not allow this.

expected to be able to do this:

```
smallX = sprand(100, 400, 0.3)
nnmfX = nnmf(smallX, 5, init=:random)
while !nnmfX.converged
    nnmfX = nnmf(smallX, 5, W=nnmfX.W, H=nnmfX.H)
end
```

### PR DESCRIPTION
**Pull request by tsano430**
This PR resolves #19.

### CODE REVIEW

**Comment by tsano430** [comment]
@ararslan, could you please review this PR?

File: README.md [hunk 1]
```diff
@@ -81,6 +81,8 @@ The function supports the following keyword arguments:
 
 - ``tol``: tolerance of changes upon convergence (default = ``1.0e-6``).
 
+- ``replicates``: Number of times to repeat factorization (default = ``1``).
```
Comment (ararslan) [inline README.md:84]: This wording makes it sound like the factorization is repeated 1 time, i.e. performed 2 times. I'd suggest this instead:
```suggestion
- ``replicates``: Number of times to perform factorization (default = ``1``).
  Further replications are not performed once the algorithm has converged.
```

**Review by ararslan — COMMENTED** [review]
The implementation looks good, just one note about the wording in the documentation. Is the idea that, rather than adjusting the maximum number of iterations and the tolerance in order to obtain convergence, you just continuing the computation where it left off to avoid repeating the computation?

**Comment by tsano430** [comment]
@ararslan Yes, you are right. I fixed README.md. Thanks!

**Comment by tsano430** [comment]
@ararslan I'm very sorry about two mistakes. I misunderstood the issue 19 and meaning of `replicates`, respectively. I'll fix the mistakes as soon as possible.

**Pull request merged**